test/unit/src/core/Object3D.tests.js

```diff
@@ -1144,6 +1144,15 @@ export default QUnit.module( 'Core', () => {
 
 		} );
 
+		QUnit.test( 'add empty array', ( assert ) => {
+
+			var obj = new Object3D();
+			obj.add(...[]);
```

Comment:
`func( ...[] )` is equivalent to `func()` for any function.

Reply:
Thanks @LeviPesin, are you suggesting I change my test to:

```js
var obj = new Object3D();
obj.add();
```

or are you suggesting that changing the current behaviour from throwing-an-error to noop is not a good idea?

Reply:
The first. About the `obj.add` - I think spread array can be used there to avoid using deprecated `arguments`?

Reply:
I.e. something like this:
```
add( ...objects ) {

    if ( objects.length !== 1) {

        for ( let object of objects ) {

            this.add( object );

         }

         return this;

    }

    const object = objects[ 0 ];
    // main code here

}
```

Reply:
Cool - I can change the test. What do you mean about `arguments` being deprecated? (I am new-ish to three.js). Should I refactor my check in `Object3D.add`?

Reply:
> What do you mean about `arguments` being deprecated?

AFAIK in ES6 and later it is recommended to use rest arguments instead the `arguments`.

> Should I refactor my check in `Object3D.add`?

That would be good!